backend/src/v2/compiler/argocompiler/common.go

```diff
@@ -99,8 +100,8 @@ func GetMLPipelineServicePortGRPC() string {
 // if CA Bundle env vars are specified.
 func ConfigureCABundle(tmpl *wfapi.Template) {
 	caBundleCfgMapName := os.Getenv("ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_NAME")
```

Comment:
might as well move `ARTIFACT_COPY_STEP_CABUNDLE_CONFIGMAP_NAME` to common as well

Reply:
Done.